### PR TITLE
Unify logging in xrdpapi/*

### PR DESCRIPTION
**Pull request by aquesnel**
Follow-on pull request to #1633 which migrates all logging in the xrdpapi directory to use the LOG() and LOG_DEVEL() macros.

Changes:

* Migrate logging to the LOG() and LOG_DEVEL() macros (using text transform script)
* error logs use the LOG macro
* update code formatting with astyle for all files in xrdp/*

### CODE REVIEW

File: xrdpapi/xrdpapi.c [hunk 1]
```diff
@@ -127,15 +122,15 @@ WTSVirtualChannelOpenEx(unsigned int SessionId, const char *pVirtualName,
 
     if (wts->display_num <= 0)
     {
-        LLOGLN(0, ("WTSVirtualChannelOpenEx: fatal error; display is 0"));
+        LOG(LOG_LEVEL_ERROR, "WTSVirtualChannelOpenEx: fatal error; display is 0");
```
Comment (matt335672) [inline xrdpapi/xrdpapi.c:125]: Should be <= 0, looking at the test.

**Review by matt335672 — COMMENTED** [review]
One minor comment.

On the whole I'm happy with this. There are definitely places where the messages themselves could be improved, but this is nothing to do with the PR, and this is in any case a bit step forward.

File: xrdpapi/xrdpapi.c (hunk 1 above)
Comment (aquesnel) [reply xrdpapi/xrdpapi.c]: done

**Comment by aquesnel** [comment]
@matt335672 I've implemented the feedback, let me know if anything else needs to be changed before this can be merged.

**Pull request merged**